opal/group.go

```diff
@@ -182,6 +182,20 @@ func resourceGroup() *schema.Resource {
 				Optional:    true,
 				Default:     true,
 			},
+			"on_call_schedule": {
+				Description: "An on call schedule for this group.",
+				Type:        schema.TypeList,
```

Comment:
Let's use `schema.TypeSet` here since the ordering doesn't matter (i.e. if I flipped the ordering of 2 schedules on my group in TF, it should not create a diff)